jujugui/static/gui/src/app/components/model-actions/_model-actions.scss

```diff
@@ -1,6 +1,11 @@
 // scss-lint:disable IdSelector
 #model-actions-container { // sass-lint:disable-line no-ids, id-name-format
-  @extend .vertical-center;
+  @media only screen and (min-width: 600px) {
```

Comment:
The mobile file contains styles that take effect at < 600px. This styling uses min-width and therefore only applies when > 600px. I didn't want to complicate the mobile file by adding desktop styles.